### PR TITLE
Aerospike Vector Search(AVS) Bedrock integration example

### PR DESCRIPTION
**Pull request by rahul-aerospike**
*Issue #, if available:*

*Description of changes:*
This pull request add example on how to use Aerospike Vector Search(AVS) with Bedrock. Please refer to [AVS page](https://aerospike.com/docs/vector) for more information.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ayanray089 — COMMENTED** [review]
1.	Please remove hard coded values of secret key, access key from the code. Instead provide proper instructions.
2.	Please provide instructions on how users can install AVS 0.10.1. Please consider that users will not be familiar with Aerospike. The notebook should be self sufficient. Do you have a cloud offering and free trial associated with it? If so, it would be better to include it instead of users going through installation.
3.	Remove hard coded values of AVS_HOST. I think it will change as per environment. Instead provide proper instructions.
4.	Please add a screenshot highlighting the context from raw data, and then print two variables 1/ response without context and 2/ response with context. This will help users to understand the value Aerospike is bringing with RAG.

File: aerospike/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Aerospike
+## Aerospike Vector Search (AVS)
+
+This directory contains pre-built samples to help customers get started with the **Generative AI** using **AWS** services and **Aerospike Vector Search**.[AVS](https://aerospike.com/docs/vector).
+
+## Get Started
+
+### Prerequisites
+
+Follow these [steps](https://aerospike.com/docs/vector/install) to setup Aerospike Vector Store cluster.
```
Comment (hev) [inline aerospike/README.md:10]:
```suggestion
If you have a license key, follow these [steps](https://aerospike.com/docs/vector/install) to setup Aerospike Vector Store cluster in your account, or you can [request a free trial](https://aerospike.com/lp/aerospike-vector-search-preview-access/) experience which provides a cloud hosted Aerospike Vector Search endpoint (AVS_HOST below). 
```

**Review by hev — CHANGES_REQUESTED** [review]
@rahul-aerospike this suggestion covers items 2 & 3. Can you help with 1 and 4?

File: aerospike/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Project setup
+
+#### Local
+
+Follow these [steps](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to install AWS CLI in your local environment.
+
+
+### Environment variables setup
+
+You can set the environment variables in the Jupyter Notebook.
```
Comment (hev) [inline aerospike/README.md:21]:
```suggestion
You can set the environment variables in the Jupyter Notebook. Setup your access key and secret key and provide below where it says `YOUR_AWS_ACCESS_KEY` and `YOUR_AWS_SECRET_KEY`
```

**Comment by rahul-aerospike** [comment]
> 4\. Please add a screenshot highlighting the context from raw data, and then print two variables 1/ response without context and 2/ response with context. This will help users to understand the value Aerospike is bringing with RAG.

Can you please further expand on the "response without context" and "response with context". Not quite sure how would I achieve it.  Can you point me to any example in the repo where the above mentioned variables are printed.

**Pull request merged**